library/headers.inc.php

```diff
@@ -70,6 +70,11 @@ function call_required_libraries($library_array){
         <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery-datetimepicker/jquery.datetimepicker.full.min.js"></script>
     <?php
     }
+    if (in_array("jquery-ui",$library_array)){   ?>
+        <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css" media="screen" />
```

Comment:
What happens if there is no internet connection? Try to work with the assets in the system, there are jquery files on the system

Reply:
Even if I put there a system file, internet connection is still needed anyway. Whats the point?